templates/default/sv-proxyd-global-run.erb

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-cd <%= node[:valhalla][:config_dir] %>
+cd <%= node[:valhalla][:conf_dir] %>
 PROXY_IN=$(jq -r ".loki.service.proxy" <%= node[:valhalla][:conf_dir] %>/<%= node[:valhalla][:config] %>)_in
```

Comment:
this has to change based on the layer. notice where it says loki, it should say the layer. maybe this:

    PROXY_IN=$(jq -r ".<%= @options[:layer] %>.service.proxy" <%= node[:valhalla][:conf_dir] %>/<%= node[:valhalla][:config] %>)_in
    PROXY_OUT=$(jq -r ".<%= @options[:layer] %>.service.proxy" <%= node[:valhalla][:conf_dir] %>/<%= node[:valhalla][:config] %>)_out